BuildRelease.msbuild

```diff
@@ -107,7 +107,7 @@
         <ItemGroup>
           <XunitTestAssemblies
               Include="Src\*Test\bin\$(Configuration)\*Test.dll"
-              Exclude="Src\AutoFixture.NUnit2.*Test\bin\$(Configuration)\*.dll" />
+              Exclude="Src\AutoFixture.NUnit*.*Test\bin\$(Configuration)\*.dll" />

```

Comment:
Do we know if this change works? See [comment](https://github.com/AutoFixture/AutoFixture/pull/528/files#r52838572) below.